static/src/javascripts/projects/commercial/modules/dfp/define-slot.js

```diff
@@ -96,6 +96,56 @@ const defineSlot = (adSlotNode: Element, sizes: Object): Object => {
         slotReady = setHighMerchSlotTargeting(slot, slotTarget);
     }
 
+    if (config.switches.iasOptimisation) {
+        console.log(`Defining ${id}`);
+        /* eslint-disable no-underscore-dangle, no-multi-assign */
+        const iasPET = (window.__iasPET = window.__iasPET || {});
+        /* eslint-enable no-underscore-dangle, no-multi-assign */
+
+        iasPET.queue = iasPET.queue || [];
+        iasPET.pubId = '10249';
+
+        let loadedResolve;
+
+        const iasDataPromise = new Promise(resolve => {
+            loadedResolve = resolve;
+        });
+
+        const iasDataCallback = () => {
+            console.log(`Applying IAS targeting: ${id}`);
```

Comment:
another console log

Reply:
As discussed in person, these are being left in to help better troubleshoot this problem on CODE.